flink-runtime/src/main/java/org/apache/flink/runtime/net/SSLUtils.java

```diff
@@ -55,6 +58,42 @@ public static boolean getSSLEnabled(Configuration sslConfig) {
 	}
 
 	/**
+	 * Sets SSl version and cipher suites for SSLServerSocket
+	 * @param socket
+	 *        Socket to be handled
+	 * @param config
+	 *        The application configuration
+	 */
+	public static void setSSLVerAndCipherSuites(ServerSocket socket, Configuration config) {
+		if (socket instanceof SSLServerSocket) {
+			((SSLServerSocket) socket).setEnabledProtocols(config.getString(
+				ConfigConstants.SECURITY_SSL_PROTOCOL,
+				ConfigConstants.DEFAULT_SECURITY_SSL_PROTOCOL).split(","));
```

Comment:
By "do explicit handing", do you mean we should check if the elements in split resutls are both legal(must be one of `SSLv3, TLSv1, TLSv1.1....`)?

BTW/FYI: This config also applies to akka, and it seems like akka only support single value setting but not multiple ones(It throws exception when I set the value to `TLSv1.1,TLSv1.2`).